src/woo_publications/metadata/api/filterset/category.py

```diff
@@ -0,0 +1,28 @@
+from django.utils.translation import gettext_lazy as _
+
+from django_filters.rest_framework import FilterSet, filters
+
+from woo_publications.api.custom_filters import URLFilter
+from woo_publications.metadata.models import InformationCategory
```

Comment:
```suggestion
from ..models import InformationCategory
```
please use relative imports for names in the same django app